src/lib/es2019.object.d.ts

```diff
@@ -0,0 +1,9 @@
+/// <reference lib="es2015.iterable" />
+
+interface ObjectConstructor {
+    /**
+     * Returns an object created by key-value entries for properties and methods
+     * @param entries An iterable object that contains key-value entries for properties and methods.
+     */
+    fromEntries(entries: Iterable<[string | symbol, any]>): any;

```

Comment:
it would be *really* ideal if this could produce an object where the values matched the `value` item in the iterable. Perhaps?:
```suggestion
    fromEntries<T = unknown>(entries: Iterable<[PropertyKey, T]>): { [k in PropertyKey]: T };
```

Reply:
Good suggestion, but a bit hesitant to do `unknown` here as no other function in lib.d.ts uses it for its return type. Would `any` be sufficient or `unknown` be required?

Reply:
Since T is inferable then any is fine, but imo nothing should ever be any, only unknown or narrower :-)

Reply:
Kinda agree but I'll go `any` for now, for consistency here...